contrib/selinux/syslog_ng.sh

```diff
@@ -11,6 +16,10 @@
 # 250: unsupported os/distribution/version
 # 249: conflicting parameters
 # 248: internal error at task selection
+# 247: internal error at callback task selection
```

Comment:
I would prefer to keep those exit codes the way they are for now. Later on, it might be worth to give it second round of thought, but at this point I don't see any added value of using variable names.

The exit codes are there so that in case of automating things, error/exit conditions can be determined more easily. (ie. running the instance from the rpm post install script.)